battleship.py

```diff
@@ -149,16 +151,29 @@ def drawGrid(data, canvas, grid, showShips):
 Returns: bool
 '''
 def isVertical(ship):
-    return
-
+    temp =[]
+    for i in range(len(ship)):
+        temp.append(ship[i][0])
+    temp.sort()
+    if ship[0][1]==ship[1][1]==ship[2][1] and temp[0]==temp[1]-1==temp[2]-2:
+        return True
+    return False
 
+  
 '''
 isHorizontal(ship)
 Parameters: 2D list of ints
 Returns: bool
 '''
 def isHorizontal(ship):
-    return
+  temp =[]
+  for i in range(len(ship)):
+        temp.append(ship[i][1])
+  temp.sort()
+  if ship[0][0]==ship[1][0]==ship[2][0] and temp[0]==temp[1]-1==temp[2]-2:
+     return True

```

Comment:
Use a loop and try to make it more generalized so that it can work for a ship of any length

Reply:
Also, resolve conflicts by deleting pycache and then clicking on the resolve conflicts button for battleship.py